web/app/models.py

```diff
@@ -245,6 +249,31 @@ class TournamentPlayer(db.Model):
     def __str__(self):
         return "Tournament Player: {}\n \
                 Member found in db: {}".format(self.name)
+
+
+class Round(db.Model):
+    __tablename__ = 'tourny_round'
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    tournament_id = db.Column(db.Integer, db.ForeignKey('tournament.id'))
+    submitted = db.Column(db.Boolean, default=False)
+
+
+class Match(db.Model):
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    player_1_aga_num = db.Column(db.Integer, db.ForeignKey('tournament_player.aga_num'), nullable=True)
```

Comment:
Do you mean to simply rename these or to revert to using the TournamentPlayer id?